src/Nethermind/Nethermind.Merge.Plugin/Handlers/V1/NewPayloadV1Handler.cs

```diff
@@ -292,14 +294,13 @@ private async Task<(ValidationResult, string? Message)> ValidateBlockAndProcess(
             }
 
             // Validate
-            bool validAndProcessed = ValidateWithBlockValidator(block, parent);
-            ValidationResult? result = ValidationResult.Syncing;
-
-            if (!validAndProcessed)
+            if (!ValidateWithBlockValidator(block, parent))
             {
                 return (ValidationResult.Invalid, string.Empty);
             }
 
+            ValidationResult? result = ValidationResult.Syncing;
```

Comment:
I assume this is just a style change?

Reply:
yes, since the above `ValidateWithBlockValidator(block, parent)` no longer needs to be stored in a variable, I changed a code style.